EMIP/1-1000/EMIP00124.xml

```diff
@@ -151,6 +152,63 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                        </layout>
                      </layoutDesc>
                   </objectDesc>
+                  <handDesc>
+                    <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="rubrication">
+                        <foreign xml:lang="gez">ሚካኤል፡</foreign>
+                        <foreign xml:lang="gez">ማርያም፡</foreign>
+                         </seg>
+                    </handNote>
+                  </handDesc>
+
+                  <decoDesc>
+                    <decoNote type="miniature" xml:id="d01"><locus target="#iv(erso)"/><desc><ref type="authFile" corresp="AT1031VirginAndChild"></ref></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d02"><locus target="#iir(ecto)"/><desc><ref type="authFile" corresp="AT1082Angel">, with <term key="sword"/> <term key="scabbard"/> </ref></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d03"><locus target="#iiv(erso)"/><desc><ref type="authFile" corresp="AT1066DavidHarp"></ref> with caption “David, King and Prophet.”</desc></decoNote>
+                    <decoNote type="miniature" xml:id="d04"><locus target="#21r"/><desc><ref type="authFile" corresp ="AT1086StrikingHead"/></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d05"><locus target="#50v"/><desc><ref type="authFile" corresp="AT1035HolyMan"> <persName ref="PRS4585Georgeo"/> and the <term key="dragon"/> </ref> with caption “Saint George.”</desc></decoNote>
+                    <decoNote type="miniature" xml:id="d06"><locus target="#51r"/><desc><ref type="authFile" corresp="AT1082Angel"> with <term key="sword"/> <term key="scabbard"/></ref></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d07"><locus target="#91r"/><desc><ref type="authFile" corresp ="AT1049Resurrection"/> with caption (l. to r.): “Eve,” and “How our Lord Jesus Christ rose from the dead on the third day.”</desc></decoNote>
+                    <decoNote type="miniature" xml:id="d08"><locus target="#117r"/><desc><ref type="authFile" corresp="AT1024MosesLaw"></ref> with caption “Prophet Moses with his tablet</desc></decoNote>
+                  </decoDesc>
+                  <additions>
+                  <list>
+                    <item xml:id="a01">
+                    <desc type="ScribalNoteCommissioning">
+                      ‎<locus target="#116v"/>
```

Comment:
these persons should be also encoded with persName and their role. See https://betamasaheft.eu/Guidelines/?id=particDescPersons , please use `<persName/>` with `@role` and `@ref`. if the person cannot be identified in the existing records, a person record should be created in consultation with the team.